include/unimplemented.h

```diff
@@ -0,0 +1,10 @@
+#ifndef UNIMPLEMENTED_H
+#define UNIMPLEMENTED_H
+
+#include <err.h>
+#include <stdlib.h>
+
+#define UNIMPLEMENTED()  \
+	errx(EXIT_FAILURE, "function %s in %s:%d not implemented", __FUNCTION__, __FILE__, __LINE__);
```

Comment:
```suggestion
	errx(EXIT_FAILURE, "function %s in %s:%d not implemented", __FUNCTION__, __FILE__, __LINE__)
```
This way we have to enforce the use of a tailing semicolon, which in return helps clang-format

Reply:
It would also be nice to have this macro in a `utils.h` file, and we could just almost always include it

Reply:
How do you feel about this @n1tram1 ?

Reply:
sounds good to me